docs/build/quick-start/hello-world.md

```diff
@@ -93,629 +87,206 @@ contract Greeter {
-    "type": "function"
-  }
-]
-```
-
-3. Set the `GREETER_CONTRACT_ABI` to require the ABI file, and add the Greeter contract address.
-
-```js
-// eslint-disable-next-line
-const GREETER_CONTRACT_ADDRESS = "0x...";
-// eslint-disable-next-line
-const GREETER_CONTRACT_ABI = require("./abi.json");
+```bash
+yarn deploy
 ```
 
-### Working with a Web3 Provider
+@tab npm
 
-1. Go to the `initializeProviderAndSigner` function in `./src/App.vue`. This function is called after the connection to Metamask is successful.
-
-In this function we should:
-
-- Initialize a `Web3Provider` and a `Signer` to interact with zkSync.
-- Initialize the `Contract` object to interact with the `Greeter` contract we just deployed.
-
-2. Import the necessary dependencies under the imports from before:
-
-```javascript
-import { Contract, Web3Provider, Provider } from "zksync-ethers";
+```bash
+npm run deploy
 ```
 
-3. Initialise the provider, signer, and contract instances like this:
-
-```javascript
-initializeProviderAndSigner() {
-    this.provider = new Provider('https://sepolia.era.zksync.dev');
-    // Note that we still need to get the Metamask signer
-    this.signer = (new Web3Provider(window.ethereum)).getSigner();
-    this.contract = new Contract(
-        GREETER_CONTRACT_ADDRESS,
-        GREETER_CONTRACT_ABI,
-        this.signer
-    );
-},
-```
+:::
 
-### Retrieving the Greeting
+You'll get the following output:
 
-Fill in the function to retrieve the greeting from the smart contract:
+```sh
+Starting deployment process of "Greeter"...
+Estimated deployment cost: 0.0000648863 ETH
 
-```javascript
-async getGreeting() {
-    // Smart contract calls work the same way as in `ethers`
-    return await this.contract.greet();
-},
-```
+"Greeter" was successfully deployed:
+ - Contract address: 0x0BaF96A7f137B05d0D35b76d59B16c86C1791D8D
+ - Contract source: contracts/Greeter.sol:Greeter
+ - Encoded constructor arguments: 0x000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000094869207468657265210000000000000000000000000000000000000000000000
 
-The function looks like this:
-
-```javascript
-initializeProviderAndSigner() {
-    this.provider = new Provider('https://sepolia.era.zksync.dev');
-    // Note that we still need to get the Metamask signer
-    this.signer = (new Web3Provider(window.ethereum)).getSigner();
-    this.contract = new Contract(
-        GREETER_CONTRACT_ADDRESS,
-        GREETER_CONTRACT_ABI,
-        this.signer
-    );
-},
-async getGreeting() {
-    return await this.contract.greet();
-},
+Requesting contract verification...
+Your verification ID is: 1781
 ```
 
-After connecting the Metamask wallet to zkSync Era Testnet, you should see the following page:
+Congratulations! You just deployed a smart contract to zkSync Sepolia testnet. You can find it in the [zkSync Sepolia explorer](https://sepolia.explorer.zksync.io/) by searching the contract address.
```

Comment:
Suggestion, to bring more visibility/cheer, should we include an emoji at the beginning?
```suggestion
🥳 Congratulations! You just deployed a smart contract to zkSync Sepolia testnet. You can find it in the [zkSync Sepolia explorer](https://sepolia.explorer.zksync.io/) by searching the contract address.
```